src/components/LoansByCategory/HelpmeChoose/HelpmeChooseRecommendations.vue

```diff
@@ -1,37 +1,19 @@
 <template>
 	<div class="tw-flex tw-flex-col lg:tw-flex-row tw-gap-3 tw-w-full">
-		<div class="tw-flex tw-flex-col tw-gap-1 lg:tw-gap-1.5">
-			<!-- eslint-disable max-len -->
-			<helpme-choose-borrower-selector
-				image-url="https://res.cloudinary.com/kiva/w_960,h_600,c_fill,g_faces/a_ignore,fl_progressive,q_auto:best,f_auto/remote/ea3be8ae591d38de82c7a81b3657b22e.jpg"
-				name="BV Goudiry Group"
-				tag="Latinx/Hispanic-Owned Business"
-				:selected="selectedLoan"
-				:index="0"
-				@select="selectLoan"
-			/>
-
-			<!-- eslint-disable max-len -->
+		<div class="tw-flex tw-flex-col tw-gap-1 lg:tw-gap-1.5 lg:tw-w-1/3">
 			<helpme-choose-borrower-selector
-				image-url="https://res.cloudinary.com/kiva/w_960,h_600,c_fill,g_faces/a_ignore,fl_progressive,q_auto:best,f_auto/remote/ea3be8ae591d38de82c7a81b3657b22e.jpg"
-				name="BV Goudiry Group"
-				tag="Agriculture"
+				v-for="(loan, index) in loans"
+				:key="loan.id"
+				:image-url="loan.image.default"
```

Comment:
Are we guaranteed that `image` and `activity` are defined for a loan? If not, we'd want computed values instead with optional chaining.

Reply:
Are we having a placeholder for default images?

Reply:
Not sure about placeholder. I've personally never seen a loan without an image. @mcstover Is this a case that we'd need to cover (no loan image), or can we assume there will be an image?

Reply:
Some components are assuming there will be an image

```
<loan-card-image
							:loan-id="loan.id"
							:name="loan.name"
							:retina-image-url="loan.image.retina"
							:standard-image-url="loan.image.default"
							:is-visitor="isVisitor"
							:is-favorite="isFavorite"
							:class="{ 'tw-rounded-tl': roundedCorners }"

							@track-loan-card-interaction="trackInteraction"
							@favorite-toggled="toggleFavorite"
						/>
```

Reply:
You could v-if a placeholder in place of the image like we do in KivaClassicBasicLoanCard

Reply:
@dyersituations some methods to have optional chaining were added

@mcstover I like the idea of having a placeholder like that but the question is if a loan can have no image?

Reply:
Loans always have a image hash assigned to them so long ad you ask for it in your query it can be assumed to resolve to an image.